sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -27,6 +27,13 @@ namespace Azure.Messaging.EventHubs
     ///   group pairing to share work by using a common storage platform to communicate.  Fault tolerance is also built-in,
     ///   allowing the processor to be resilient in the face of errors.
     /// </summary>
+    ///
+    /// <remarks>
+    ///   The <see cref="EventProcessorClient" /> is safe to cache and use for the lifetime of an application, and that is best practice for use when the application
```

Comment:
Did you mean to say this?
```suggestion
    ///   The <see cref="EventProcessorClient" /> is safe to cache and use for the lifetime of an application, and that is best practice to use when the application
```

Reply:
No, I meant "for use" ... but comparing that to your suggestion, I think yours reads better.   I'll update with the change.

Reply:
On re-read, I think that we can cut a couple of words.   Going with - "... of an application, and that is best practice when the ..."